plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -1,4 +1,10 @@
-(function(){
+/**
+ * A DOM element.
+ * @external Element
```

Comment:
This isn't necessary. You can just use `Element` from TypeScript's DOM lib. (See the `getSettings` version [here](https://github.com/PrismJS/prism/pull/2723/files#r560946722).)

Reply:
Got it. I just intended to separate the ECMAScript types (`Number`, `String`, `Object`) from the DOM types.